apps/contrib/translations.py

```diff
@@ -1,73 +1,29 @@
-from django.db import models
 from django.utils import translation
-from wagtail.core.models import PageBase
-
-from digitalstrategie.settings import LANGUAGES
+from wagtail.core.blocks.stream_block import StreamValue
 
 
 class TranslatedField(object):
-    fallback_lang = 'de'
-
-    def __init__(self, field_name, field, overwrite_fallback={}):
-        for language_code, language in LANGUAGES:
-            setattr(self, language_code + '_field',
-                    field_name + '_' + language_code)
-        self.field_name = field_name
-        self.field = field
-        self.overwrite_fallback = overwrite_fallback
 
-    def __get__(self, instance, owner):
-        lang_code = translation.get_language()
-        lang_field_name = self._get_translated_field_name(lang_code)
-        value = getattr(instance, lang_field_name)
-
-        if value:
-            return value
-        else:
-            field_name = self._get_translated_field_name(self.fallback_lang)
-            return getattr(instance, field_name)
+    def __init__(self, de_field, en_field):
+        self.de_field = de_field
+        self.en_field = en_field
 
-    def _get_translated_field_name(self, lang_code):
-        return '{}_{}'.format(self.field_name, lang_code)
-
-    def _get_model_fields_spec(self):
-        translated_attrs = {}
-        for lang_code, lang_name in LANGUAGES:
-            if lang_code == self.fallback_lang and self.overwrite_fallback:
-                name, path, args, kwargs = self.field.deconstruct()
-                kwargs.update(self.overwrite_fallback)
-                field = self.field.__class__(*args, **kwargs)
+    def hasContent(self, field):
+        if isinstance(field, StreamValue):
+            return len(field) != 0
```

Comment:
This line I had to adjust, because apparently there is a change in wagtail 2.14, see [here](https://githubmemory.com/repo/infoportugal/wagtail-modeltranslation/issues/346). I then copied from their code [here](https://github.com/infoportugal/wagtail-modeltranslation/blob/c0c3788628b9d6c5bb3f6c3d06dbfd483c2b2a2a/wagtail_modeltranslation/patch_wagtailadmin.py#L578) and this works now. We might have to do that also in other projects when we update wagtail..